crypto/encrypt.go

```diff
@@ -13,6 +13,10 @@ import (
 	"golang.org/x/crypto/twofish"
 )
 
+const (
+	TwofishOverhead = 28 // number of bytes added by EncryptBytes
```

Comment:
should probably be `EncryptOverhead`, it's the GCM that's causing overhead, not Twofish.